### PR TITLE
Fixes zero width or height image reading

### PR DESCRIPTION
**Pull request by flippmoke**
Updated for a bug associated with mapnik-vector-tile. Initial commit points to a branch on mapnik-vector-tile, eventually will point to released version. 

/cc @springmeyer for review.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 3.6.1
+
+Updated for a fix associated with mapnik-vector-tile where images could be requested that would have a width or height of zero resulting
+in exceptions.
+
+- Updated to mapnik-vector-tile@1.3.0
```
Comment (springmeyer) [inline CHANGELOG.md]: Did you mean `1.3.1`?

**Comment by springmeyer** [comment]
@flippmoke - I presume the next step here is to revisit this after the changes to mapnik core? https://github.com/mapnik/mapnik/pull/3672

**Comment by flippmoke** [comment]
@springmeyer yes, the changes here will be tweaked slightly once we have an official mapnik core release.

**Comment by flippmoke** [comment]
Waiting on binaries to be built for Mapnik core on windows and then hopefully we are good to merge this and start release process.

**Pull request merged**